src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -563,7 +563,8 @@ private void setupAutoCompletion() {
             suggestionProviders = new SuggestionProviders(getDatabase(), Globals.journalAbbreviationRepository, autoCompletePreferences);
         } else {
             // Create empty suggestion providers if auto-completion is deactivated
```

Comment:
Delete the commented as it's wrong and also the commented out code

Reply:
Removed those in new commit